.gitignore

```diff
@@ -1,4 +1,4 @@
 venv/
 *.pyc
-trackside_ops/settings.py
+/trackside_ops/settings.py
```

Comment:
@lealal This looks weird, does it work?

Reply:
No, and the other one didn't work either. Apparently you can add settings.py to .gitignore lol I've seen people doing settings.py and local_settings.py